### PR TITLE
Generate docker image

### PR DESCRIPTION
**Pull request by eamagdalena**
We need that S2S generates a docker images that gets stored on Azure Container Registry so other apps using it can docker compose it.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8-jre
+FROM hmcts/cnp-java-base:openjdk-jre-8-alpine-1.4
```
Comment (timja) [inline Dockerfile]: isn't more configuration needed for this?
https://github.com/hmcts/cmc-claim-store/pull/449/files#diff-3254677a7917c6c01f55212f86c57fbfR3

Comment (eamagdalena) [reply Dockerfile]: Fixed 🗡

File: Dockerfile [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
 
 RUN mkdir -p /opt/app
 
-COPY build/install/service-auth-provider /opt/app
+COPY build/libs/$APP /opt/app/
 
 EXPOSE 8489
 
-ENTRYPOINT [ "/opt/app/bin/service-auth-provider" ]
+ENTRYPOINT exec java ${JAVA_OPTS} -jar "/opt/app/service-auth-provider.jar"
```
Comment (timja) [inline Dockerfile]: isn't this entrypoint not needed anymore?

Comment (eamagdalena) [reply Dockerfile]: In the build is failing to find it on the copy command so i guess it has been broken for a while

**Pull request merged**